Jenkinsfile

```diff
@@ -945,6 +937,46 @@ pipeline {
                         }
                     }
                 } // stage('Scan CentOS 7 RPMs')
+                stage('Fault injection testing') {
+                    when {
+                        beforeAgent true
+                        expression { ! skipStage() }
+                    }
+                    agent {
+                        dockerfile {
```

Comment:
Since this runs in Docker, any reason why it cannot be a GHA test?

Reply:
Good question, I think you could run it but it would take a lot longer and we wouldn't be able to use the warnings-ng plugin to check for regressions.

Reply:
GHA has something called _Problem Matchers_ which are being used to surface compiler warnings, etc. similar to what warnings-ng does, IIUC.  I started my introduction to them at https://discuss.python.org/t/using-github-problem-matchers-to-catch-warnings-early/4254

As you are the primary warnings-ng user/implementer, what are your thoughts on *Problem Matcher*s?

Reply:
Different but related feature.

We already use problem matchers both in the doxygen action where I wrote my own, and in NLT https://github.com/daos-stack/daos/blob/master/utils/node_local_test.py#L297-L301 where we print output to match what github already searches for.

The warnings-ng feature that we use in Jenkins is that of taking the delta from one build to another, and notifying or failing the build based on regressions.